scripts/extract_dts_includes.py

```diff
@@ -415,24 +425,39 @@ def extract_property(yaml, node_address, y_key, y_val, names, prefix, defs):
     extract_cells(node_address, yaml, y_key,
                   names, 0, prefix, defs, def_label)
   else:
-    extract_single(node_address, yaml[get_compat(reduced[node_address])],
+    extract_single(node_address, yaml,
                    reduced[node_address]['props'][y_key], y_key,
                    prefix, defs, def_label)
 
   return
 
-def extract_node_include_info(reduced, node_address, yaml, defs, structs):
-  node = reduced[node_address]
-  node_compat = get_compat(node)
+def extract_node_include_info(reduced, root_node_address, sub_node_address, yaml, defs, structs, y_sub):
+  node = reduced[sub_node_address]
+  node_compat = get_compat(reduced[root_node_address])
+  label_override = None
 
   if not node_compat in yaml.keys():
     return {}, {}
 
-  y_node = yaml[node_compat]
+  if y_sub == None:
+    y_node = yaml[node_compat]
+  else:
+    y_node = y_sub
+
+  if yaml[node_compat].get('use-property-label', False) != False:
```

Comment:
PEP008: "Don't compare boolean values to True or False using == ."

(I.e., `if if yaml[node_compat].get('use-property-label', False):`.)